### ISSUE TITLE
Charts tokens - dupes in dark CSS

### ISSUE THREAD

**Issue by mcoker** [patternfly/design-tokens#53]
Looks like all of the charts tokens are output again in [tokens-charts-dark.scss](https://github.com/patternfly/design-tokens/blob/main/packages/module/build/css/tokens-charts-dark.scss), not just the tokens that are different for dark theme compared to [tokens-charts.scss](https://github.com/patternfly/design-tokens/blob/main/packages/module/build/css/tokens-charts.scss). Shouldn't the dark stylesheet only include necessary overrides to the default tokens?

**Comment by srambach** [comment]
I agree, the dark charts stylesheet should only include the overrides.

At a glance, it seems to be something with the structure of the collection in figma and/or the exporter - the dark charts tokens aren't coming with 'dark' as a layer.

Once that's done, the dark charts config might also need to look for `charts.dark` instead of just `charts`
